config.yaml

```diff
@@ -103,10 +103,17 @@ params:
   # This menu appears only if you have at least one [params.versions] set.
   version_menu: Versions
   latest_stable_version: v3.5
+  version:
+
+    deprecation_warning: >-
+      This version of etcd is no longer supported. For the latest version, please see the [latest stable version](/docs/latest/).
   versions:
     - version: v3.6
       url: /docs/v3.6/
 
+    - version: latest
```

Comment:
Hey @moficodes thanks for tackling this.  The deploy preview is looking good, however can you explain why we need this new version entry added?

I thought `docs/latest` and `docs/next` where meta versions that we manage as redirects in https://github.com/etcd-io/website/blob/bdb510e276299794734e08ddd96f0cb2686bbc59/layouts/index.redirects#L1-L12

Reply:
A local change I fotgot to remove.
Removed now!

Reply:
Hmm one other issue, unfortunately the banner is showing on 3.4 https://deploy-preview-900--etcd.netlify.app/docs/v3.4/

3.4 is definitely still supported so we need to tweak the logic in the partial to address I think